lib/index.js

```diff
@@ -178,6 +178,35 @@ function _ensure_cheerio(value) {
     return value;
 }
 
+class DictionaryAsset extends BaseAsset {
+    constructor() {
+        super();
+        this._object_type = "DictionaryObject";
+        this._content_type = "text/html";
+    }
+
+    set_word (value) { this._word = value; }
+    set_wordDefinition (value) { this._wordDefinition = value; }
```

Comment:
@ptomato thanks! please continue to drive/walk/run-by any code of mine and offer review.